docs/source/executing-operations/subscription-callback-protocol.mdx

```diff
@@ -52,6 +53,7 @@ All actions described in these steps are performed by one of **Router**, **Subgr
     - `callback_url`: The URL that **Emitter** will send subscription data to
     - `subscription_id`: The generated unique ID for the subscription operation
     - `verifier`: A string that **Emitter** will include in all HTTP callback requests to verify its identity
+    - `heartbeat_interval_ms`: The number of milliseconds a heartbeat has to be sent to the callback endpoint for the subscription operation
```

Comment:
I think you should call out that this is optional? i.e. if not specified then we don't send heartbeat?

Reply:
Oh no you're right I didn't document the behavior when heartbeat is disabled. The logic I wrote is when the heartbeat is set to 0 it means it's disabled